pre_commit/commands/run.py

```diff
@@ -396,3 +402,12 @@ def run(
 
     # https://github.com/python/mypy/issues/7726
     raise AssertionError('unreachable')
+
+def run(*args: Any) -> int:
```

Comment:
Ug, I hate that Python/mypy doesn't have a better way to do this. I hate losing type info like this (though it almost certainly won't lead to an issue here).

Dunno if it matters, but we might want to also forward`**kwargs`? Always seems a bit odd to me to implicitly disallow kwargs using this pattern.

Alternatively we could just copy the args, which I honestly don't think is too bad, but it's not a big deal either way.

Reply:
Yeah, I should just copy the args here -- changed.